05-angularjs/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<link rel="stylesheet" type="text/css" href="node_modules/bootstrap/dist/css/bootstrap.min.css" />			
+		<link rel="stylesheet" type="text/css" href="css/style.css" />
+
+	</head>
+	<body ng-app="moviesApp">
+		<header>
+			<ul class="nav nav-tabs">
+				<li role="presentation"><a href="#/">Home</a></li>			
+			</ul>
+		</header>
+
+		<section>
+			<div ng-view></div>
+		</section>
+		
+	
+	<script type="text/javascript" src="node_modules/angular/angular.min.js"></script>
+	<script type="text/javascript" src="node_modules/angular-route/angular-route.min.js"></script>
+	<script type="text/javascript" src="js/app.js"></script>
+	<script type="text/javascript" src="node_modules/jquery/dist/jquery.min.js"></script>
```

Comment:
Do you really need jQuery? Angular has a jQuery lite built-in, if you need to do basic jQuery stuff...